httpclient-backend/fs2/src/main/scala/sttp/client3/httpclient/fs2/HttpClientFs2Backend.scala

```diff
@@ -72,61 +73,56 @@ class HttpClientFs2Backend[F[_]: ConcurrentEffect: ContextShift] private (
 
   override protected def standardEncoding: (Stream[F, Byte], String) => Stream[F, Byte] = {
     case (body, "gzip")    => body.through(fs2.compression.gunzip()).flatMap(_.content)
-    case (body, "deflate") => body.through(fs2.compression.inflate())
+    case (body, "deflate") => body.through(fs2.compression.inflate(InflateParams.DEFAULT))
     case (_, ce)           => Stream.raiseError[F](new UnsupportedEncodingException(s"Unsupported encoding: $ce"))
   }
 }
 
 object HttpClientFs2Backend {
   type Fs2EncodingHandler[F[_]] = EncodingHandler[Stream[F, Byte]]
 
-  private def apply[F[_]: ConcurrentEffect: ContextShift](
+  private def apply[F[_]: Async: Dispatcher](
       client: HttpClient,
-      blocker: Blocker,
       closeClient: Boolean,
       customizeRequest: HttpRequest => HttpRequest,
       customEncodingHandler: Fs2EncodingHandler[F]
   ): SttpBackend[F, Fs2Streams[F] with WebSockets] =
     new FollowRedirectsBackend(
-      new HttpClientFs2Backend(client, blocker, closeClient, customizeRequest, customEncodingHandler)
+      new HttpClientFs2Backend(client, closeClient, customizeRequest, customEncodingHandler)
     )
 
-  def apply[F[_]: ConcurrentEffect: ContextShift](
-      blocker: Blocker,
+  def apply[F[_]: Async: Dispatcher](
       options: SttpBackendOptions = SttpBackendOptions.Default,
       customizeRequest: HttpRequest => HttpRequest = identity,
       customEncodingHandler: Fs2EncodingHandler[F] = PartialFunction.empty
   ): F[SttpBackend[F, Fs2Streams[F] with WebSockets]] =
     Sync[F].delay(
       HttpClientFs2Backend(
         HttpClientBackend.defaultClient(options),
-        blocker,
         closeClient = true,
         customizeRequest,
         customEncodingHandler
       )
     )
 
-  def resource[F[_]: ConcurrentEffect: ContextShift](
-      blocker: Blocker,
+  def resource[F[_]: Async: Dispatcher](
```

Comment:
same, remove `Dispatcher` and create it here in `Resource`

Reply:
https://github.com/softwaremill/sttp/pull/738/commits/e93160fc787f6ddee9797349642bc4bad94539d7